examples/jsm/nodes/accessors/TextureNode.js

```diff
@@ -136,7 +138,15 @@ class TextureNode extends UniformNode {
 
 			}
 
-			return builder.format( propertyName, nodeType, output );
+			let snippet = propertyName;
+
+			if ( builder.needsColorSpace( this.value ) ) {
```

Comment:
I think we should name this method something like `needsForceLinear` or `needsConversionToLinear`?

Reply:
Hmm.. Maybe needsColorConversionToLinear